Makefile

```diff
@@ -119,7 +118,12 @@ clean: clean_docs
 	rm -rf $(BUILD_DIR)
 	rm -f $(GOPATH)/bin/crc
 	rm -rf $(RELEASE_DIR)
-       
+
+.PHONY: build_integration
+build_integration: $(SOURCES)
+	GOOS=linux   go test ./test/integration/ -c -o $(BUILD_DIR)/linux-amd64/integration.test
+	GOOS=windows go test ./test/integration/ -c -o $(BUILD_DIR)/windows-amd64/integration.test.exe
+	GOOS=darwin  go test ./test/integration/ -c -o $(BUILD_DIR)/darwin-amd64/integration.test
```

Comment:
@guillaumerose This is really new for me, I didn't know we can even do that :(. can this binary used directly starting the integration tests with the arguments?

Reply:
@guillaumerose Also for mac target we use `macos-amd64` instead `darwin-amd64`

Reply:
Fixed macos/darwin mistake.

Yes we can use the binary! The only issue I have is that it uses `os.Getwd` and it really wants to be launched in `test/integration`.

The Makefile should look like

``` 
integration: 
	(cd ./test/integration && $(BUILD_DIR)/linux-amd64/integration.test --test.timeout=120m -test.v $(PULL_SECRET_FILE) $(BUNDLE_LOCATION) $(CRC_BINARY) --bundle-version=$(BUNDLE_VERSION) $(GODOG_OPTS))
```

I will open a new PR with this change, if it's ok for you :)

Reply:
Ah so it is only create the binary which doesn't include feature files and that's the reason needed to launched from `test/integration` ?

Reply:
yes!